Apigee/ManagementAPI/Developer.php

```diff
@@ -450,7 +450,7 @@ public function save($force_update = false, $old_email = null)
         // We must also do this when creating a developer ex nihilo in order
         // to set initial status. Otherwise new developer will have default
         // status, which is generally 'approved'.
-        $this->post($old_email . '?action=' . $cached_status);
+        $this->post($this->email . '?action=' . $cached_status);
```

Comment:
This was discovered when we had a request from NTT.com to have the developer email editable, which is tracked under DEVSOL-1013.

So when update the developer email $old_email is no more available and doing a POST to $old_email fails.

But if we do a POST to $this->email it will be the updated email.

This issue is tracked under DEVSOL-1291